extensions/resource-deployment/src/interfaces.ts

```diff
@@ -171,9 +172,17 @@ export type ComponentCSSStyles = {
 	[key: string]: string;
 };
 
+export interface IOptionsSource {
+	readonly type: OptionsSourceType,
+	readonly variableNames: { [index: string]: string; },
+	getOptions(): Promise<string[]>,
```

Comment:
it could also be azdata.categoryvalue[], right?

Reply:
fixed